apps/language_server/lib/language_server/server.ex

```diff
@@ -530,13 +530,20 @@ defmodule ElixirLS.LanguageServer.Server do
   end
 
   defp handle_request(code_lens_req(_id, uri), state) do
-    if dialyzer_enabled?(state) and state.settings["suggestSpecs"] != false do
-      {:async,
-       fn -> CodeLens.code_lens(state.server_instance_id, uri, state.source_files[uri].text) end,
-       state}
-    else
-      {:ok, nil, state}
+    fun = fn ->
+      {:ok, spec_code_lens} =
```

Comment:
use with or case to handle/return error, original code returned error, your will crash